src/plugins/plot/configuration/YAxisModel.js

```diff
@@ -57,7 +57,14 @@ export default class YAxisModel extends Model {
         this.listenTo(this, 'change:logMode', this.onLogModeChange, this);
         this.listenTo(this, 'change:frozen', this.toggleFreeze, this);
         this.listenTo(this, 'change:range', this.updateDisplayRange, this);
-        this.updateDisplayRange(this.get('range'));
+        const range = this.get('range');
+        this.updateDisplayRange(range);
+        //This is an edge case and should not happen
+        const invalidRange = !range || (range?.min === undefined || range?.max === undefined);
```

Comment:
nitpick: we can remove the `?` from `range?.min` and `range?.max` here since we know range is not null or undefined by that point

Reply:
Done